cli/podman.adoc

```diff
@@ -104,16 +104,37 @@ podman
 . Create a Skupper site:
 +
 --
-The simplest Skupper site allows you to link to other sites, but does not support linking _to_ the current site.
+
+Use the following command to create a site where tokens are created to link on any network interface:
 
 [source, bash]
 ----
 $ skupper init
+----
 
+The following output is displayed:
+
+[source, bash]
+----
 It is recommended to enable lingering for <username>, otherwise Skupper may not start on boot.
 Skupper is now installed for user '<username>'.  Use 'skupper status' to get more information.
 ----
 
+Use the following command to start the site service at system start and persist over logouts:
+
+[source, bash]
+----
+# loginctl enable-linger <username>
+----
+
+By default, `skupper init` tries to include all IP addresses associated with local network interfaces as valid ingress hosts.
+You can use `--ingress-host <IP/Hostname>` to restrict token ingress to a specific network context:
+
+[source, bash]
+----
+$ skupper init --ingress-host my-cloud-vm.example.com
```

Comment:
It is better. WDYT about adding a note that `--ingress-host` can also be used when you create a token?
This way, if someone has multiple ingress hosts, they can specify exactly which host will be defined in
the token, when the token is created.

Reply:
By the way, this could be handled separately if you find appropriate.

Reply:
I think that should be handled in token section, but good point to check that out, I've made changes there and pinged you